app/controllers/api/v1/orders_controller.rb

```diff
@@ -26,6 +26,7 @@ class OrdersController < Api::V1::ApiController
           param :beneficiary_id, String
           param :address_id, String
           param :booking_type_id, String, desc: 'Booking type.(Online order or appointment)'
+          param :staff_note, String, desc: 'Notes for internal use'
```

Comment:
how come we have those changes here ? I added that for the sticky notes (already merged) 🤔

Reply:
Yeah it's weird. They appeared when I rebased and merged from master. I'll replay my commits on another branch to clean it up.